src/AccessibilityInsights.SharedUx/Properties/Resources.resx

```diff
@@ -501,16 +501,16 @@ Accessibility Insights Support Team
     <value>Instructions</value>
   </data>
   <data name="ColorContrast_HowToTestAuto" xml:space="preserve">
-    <value>To auto detect an element's contrast ratio, hover over the element within a target application or set keyboard focus on the element. The analyzer will update with the auto detected colors, the contrast ratio, and the confidence level.</value>
+    <value>Auto-detect an element’s color contrast ratio by hovering over, or keyboard focusing on an element in your target application.</value>
```

Comment:
There [shouldn't](https://owl.purdue.edu/owl/general_writing/punctuation/commas/extended_rules_for_commas.html) be a comma here:
```suggestion
    <value>Auto-detect an element’s color contrast ratio by hovering over or keyboard focusing on an element in your target application.</value>
```